xbmc/cores/VideoPlayer/DVDCodecs/DVDCodecUtils.cpp

```diff
@@ -12,6 +12,8 @@
 #include "cores/FFmpeg.h"
 #include "cores/VideoPlayer/Interface/TimingConstants.h"
 
+#include <array>
```

Comment:
I think Util.h can be removed in this file now.
Trying to look at the others, but its a bit of a mixed bag in that header, haha.

Reply:
updated, thanks!